drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -16,19 +16,19 @@ class HmisSchema::QueryType < Types::BaseObject
     include Types::HmisSchema::HasOrganizations
     include Types::HmisSchema::HasClients
 
-    projects_field :projects, type: Types::HmisSchema::Project.page_type, description: 'Get a list of projects'
+    projects_field :projects
 
     def projects(**args)
       resolve_projects(Hmis::Hud::Project.all, **args)
```

Comment:
This is now used in place of `project_search` query. The `HasProjects` resolver handles the search term and the viewability filter.